sky/templates/kubernetes-ray.yml.j2

```diff
@@ -244,6 +244,28 @@ provider:
             selector:
                 component: {{cluster_name_on_cloud}}-head
 
+    {%- if pvcs %}
+    pvcs:
+    {%- for pvc in pvcs %}
+    - apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: {{pvc.name}}
+        namespace: {{k8s_namespace}}
+      spec:
+        accessModes:
+          - {{pvc.access_mode}}
+        {%- if pvc.storage_class_name %}
+        storageClassName: {{pvc.storage_class_name}}
+        {%- endif %}
+        resources:
+          requests:
+            storage: {{pvc.disk_size}}Gi
+      autoDelete: {{pvc.auto_delete}}
```

Comment:
nit: I noticed we use `auto_delete/autoDelete` internally for Volumes. Can we use persistent to be consistent with the task YAML?